birdhouse/components/finch/service-config.json.template

```diff
@@ -20,6 +24,11 @@
       "rel": "service-desc",
       "type": "text/xml",
       "href": "https://${PAVICS_FQDN_PUBLIC}${TWITCHER_PROTECTED_PATH}/finch?service=WPS&request=GetCapabilities"
+    },
+    {
+      "rel": "service-meta",
+      "type": "application/vnd.oci.image.index.v1+json",
+      "href": "${FINCH_IMAGE_URI}"
```

Comment:
Just my 2 cents, since the URI, do not start with http, should we name the new field "uri" instead of "href"?  "href" would suggest starting with http usually.  Ignore if it has to be "href" for compat or other reasons.

Reply:
`"href"` is required by the schema so we can't change the name

Reply:
A non-http scheme value is a valid URI. It uniquely identifies the resource. 
I would prefer to have a `docker://` scheme or similar to be more explicit if permitted, but docker does not seem to like it. However, the values provided in the defaults can be used directly (e.g. `docker pull 'registry.hub.docker.com/pavics/weaver:5.0.0'` works, but `http://registry.hub.docker.com/pavics/weaver:5.0.0` doesn't), so I find them more useful although maybe counter-intuitive.

Maybe one way to avoid confusion would be to use `index.docker.io/pavics/weaver:5.0.0` instead, since it does not redirect to `https://hub.docker.com/` like `registry.hub.docker.com` does? This alternate location still woks with `docker pull 'index.docker.io/pavics/weaver:5.0.0'`, and it looks slightly more like the `type` definition. The idea of this link is really to have the exact value that can be plugged in the docker-compose service `image` field.